SassHandler.php

```diff
@@ -203,15 +203,14 @@ class SassHandler extends CApplicationComponent
      */
     public function init()
     {
-        $vendorPath = __DIR__ . '/../../';

```

Comment:
Ok, i understand. We are saying the same think, but i was thinking of artem-frolov/yii-sass to be on the extension folder because i do it in the manual way.